src/templates/sdk/SdkReference.tsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react'
+import Layout from '../../components/Layout'
+import SEO from '../../components/seo'
+import PostLayout from '../../components/PostLayout'
+import CommunityQuestions from '../../components/CommunityQuestions'
+import { docsMenu } from '../../navs'
+import ReactMarkdown from 'react-markdown'
+import Accordion from '../../components/SdkReferences/Accordion'
+import Parameters from '../../components/SdkReferences/Parameters'
+import FunctionReturn from '../../components/SdkReferences/Return'
+import FunctionExamples from '../../components/SdkReferences/Examples'
+import CopyMarkdownActionsDropdown from '../../components/MarkdownActionsDropdown'
+import { useLocation } from '@reach/router'
+import Link from '../../components/Link'
+import { getLanguageFromSdkId } from '../../components/SdkReferences/utils'
+
+interface Parameter {
+    name: string
+    type: string
+    description: string
+    isOptional: boolean
+}
+
+interface Example {
+    id: string
+    name: string
+    code: string
+}
+
+interface SdkFunction {
+    id: string
+    title: string
+    description: string
+    category: string
+    showDocs: boolean
+    examples: Example[]
+    params: Parameter[]
+    returnType: {
+        id: string
+        name: string
+    }
+    returnDescription?: string
+    details: string
+    releaseTag?: string
+}
+
+interface Class {
+    id: string
+    title: string
+    description: string
+    functions: SdkFunction[]
+}
+
+interface SdkReferenceData {
+    id: string
+    hogRef: string
+    noDocsTypes: string[]
+    info: {
+        description: string
+        id: string
+        slugPrefix: string
+        specUrl: string
+        title: string
+        version: string
+    }
+    classes: Class[]
+}
+
+interface PageContext {
+    fullReference: SdkReferenceData
+    types: string[]
+}
+
+const padDescription = (description: string): string => {
+    return description.replace(/\n/g, '\n\n')
```

Comment:
Markdown needs a double next line to render paragraphs properly. We don't always get these depending on how the doc comments are provided.